src/pages/mainnet-renderer/[componentAccountId]/widget/[componentName].tsx

```diff
@@ -0,0 +1,77 @@
+import * as nearAPI from 'near-api-js';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+
+import { VmComponent } from '@/components/vm/VmComponent';
+import { useDefaultLayout } from '@/hooks/useLayout';
+import type { NextPageWithLayout } from '@/utils/types';
+
+const ViewComponentPage: NextPageWithLayout = () => {
+  const router = useRouter();
+  const componentSrc = router.query.componentAccountId ? `${router.query.componentAccountId}/widget/${router.query.componentName}` : null;
+  const [ componentCode, setComponentCode ] = useState('return <div>loading mainnet renderer..</div>')
+
+  useEffect(() => {
+    const getComponent = async (componentPath: string | null) => {
+      if(!componentPath) return;
+      const keyPair = nearAPI.KeyPair.fromRandom('ed25519');
+      const keyStore = new nearAPI.keyStores.InMemoryKeyStore();
+      const nearConfig = {
+        networkId: 'testnet',
```

Comment:
I'm confused why this route is called the `mainnet-renderer` while this is hardcoded to `testnet`. What am I missing here?

Reply:
The code that we will be running as mainnet code will be fetched from `testnet` SocialDB so we need to use testnet near config and rpc. https://github.com/near/near-discovery/pull/225#discussion_r1244534813 describes whats happening with the code we fetch